### PR TITLE
Mobile responsive page common components

### PR DESCRIPTION
**Pull request by onkar-saeloun**
What-
PR includes breakdown changes from [this](mobile-responsive-personal-details-page) PR

why- 
For better review breaks down changes in different files.

### CODE REVIEW

File: app/javascript/src/common/CustomAsyncSelect/index.tsx [hunk 1]
```diff
@@ -4,9 +4,9 @@ import React from "react";
 
 import AsyncSelect from "react-select/async";
 
-import { customErrStyles } from "common/CustomReactSelect/CustomErrStyle";
-import { customStyles } from "common/CustomReactSelect/CustomStyle";
 import { CustomValueContainer } from "common/CustomReactSelect/CustomValueContainer";
+import { customErrStyles } from "common/CustomReactSelectErrStyles";
+import { customStyles } from "common/CustomReactSelectStyles";
```
Comment (prasanthchaduvula) [inline app/javascript/src/common/CustomAsyncSelect/index.tsx:9]: Please make these import statements in the following way and export all the components related to `customReactSelect` from `customReactSelect `itself.
`import { CustomValueContainer, customErrStyles, customStyles } from "common/CustomReactSelect";`

File: app/javascript/src/common/Mobile/MobileDetailsHeader/index.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import { X } from "phosphor-react";
```
Comment (prasanthchaduvula) [inline app/javascript/src/common/Mobile/MobileDetailsHeader/index.tsx:3]: Import icons from `miruIcons` only

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

File: app/javascript/src/common/CustomAsyncSelect/index.tsx (hunk 1 above)
Comment (onkar-saeloun) [reply app/javascript/src/common/CustomAsyncSelect/index.tsx:9]: @prashanth, `CustomReactSelect` component should have only react select code. There is no point to keep these independent things inside it. Can you tell me what is your thinking on this?

File: app/javascript/src/common/Mobile/MobileDetailsHeader/index.tsx (hunk 2 above)
Comment (onkar-saeloun) [reply app/javascript/src/common/Mobile/MobileDetailsHeader/index.tsx:3]: okay

File: app/javascript/src/common/CustomAsyncSelect/index.tsx (hunk 1 above)
Comment (onkar-saeloun) [reply app/javascript/src/common/CustomAsyncSelect/index.tsx]: Exported from one file

**Review by prasanthchaduvula — APPROVED** [review]

**Pull request merged**